From 38974037a426a03351595fdcd82c0c5afc66321c Mon Sep 17 00:00:00 2001
From: Inventor1938 <ivan.kolev@mail.bg>
Date: Wed, 27 Feb 2019 12:13:41 +0200
Subject: [PATCH 1/3] Revert "PowerManagerService: Fix updating of
 mUserActivitySummary"

This reverts commit 84d7cabc7d8100564cde22c03e76c467d47f0f08.

Conflicts:
	services/core/java/com/android/server/power/PowerManagerService.java

Change-Id: I86fad2eb09ff095ef46b34874641837968e8d5a7
---
 .../server/power/PowerManagerService.java     | 42 +------------------
 1 file changed, 2 insertions(+), 40 deletions(-)

diff --git a/services/core/java/com/android/server/power/PowerManagerService.java b/services/core/java/com/android/server/power/PowerManagerService.java
index eaaa77600bcd..ebf83e0d5774 100644
--- a/services/core/java/com/android/server/power/PowerManagerService.java
+++ b/services/core/java/com/android/server/power/PowerManagerService.java
@@ -2084,57 +2084,19 @@ public final class PowerManagerService extends SystemService
                             + screenOffTimeout - screenDimDuration;
                     if (now < nextTimeout) {
                         mUserActivitySummary = USER_ACTIVITY_SCREEN_BRIGHT;
-                        if (mWakefulness == WAKEFULNESS_AWAKE) {
-                            int buttonBrightness;
-                            if (mHardwareKeysDisable) {
-                                buttonBrightness = 0;
-                            } else {
-                                if (mButtonBrightnessOverrideFromWindowManager >= 0) {
-                                    buttonBrightness = mButtonBrightnessOverrideFromWindowManager;
-                                } else {
-                                    buttonBrightness = mButtonBrightness;
-                                }
-                            }
-                            mLastButtonActivityTime = mButtonBacklightOnTouchOnly ?
-                                    mLastButtonActivityTime : mLastUserActivityTime;
-                            if (mButtonTimeout != 0
-                                    && now > mLastButtonActivityTime + mButtonTimeout) {
-                                mButtonsLight.setBrightness(0);
-                                mButtonOn = false;
-                            } else {
-                                if ((!mButtonBacklightOnTouchOnly || mButtonPressed) &&
-                                        !mProximityPositive) {
-                                    mButtonsLight.setBrightness(buttonBrightness);
-                                    mButtonPressed = false;
-                                    if (buttonBrightness != 0 && mButtonTimeout != 0) {
-                                        mButtonOn = true;
-                                        if (now + mButtonTimeout < nextTimeout) {
-                                            nextTimeout = now + mButtonTimeout;
-                                        }
-                                    }
-                                } else if (mButtonBacklightOnTouchOnly && mButtonOn &&
-                                        mLastButtonActivityTime + mButtonTimeout < nextTimeout) {
-                                    nextTimeout = mLastButtonActivityTime + mButtonTimeout;
-                                }
-                            }
-                        }
                     } else {
                         nextTimeout = mLastUserActivityTime + screenOffTimeout;
                         if (now < nextTimeout) {
                             mUserActivitySummary = USER_ACTIVITY_SCREEN_DIM;
-                            if (mWakefulness == WAKEFULNESS_AWAKE) {
-                                mButtonsLight.setBrightness(0);
-                                mButtonOn = false;
-                            }
                         }
                     }
                 }
-
                 if (mUserActivitySummary == 0
                         && mLastUserActivityTimeNoChangeLights >= mLastWakeTime) {
                     nextTimeout = mLastUserActivityTimeNoChangeLights + screenOffTimeout;
                     if (now < nextTimeout) {
-                        if (mDisplayPowerRequest.policy == DisplayPowerRequest.POLICY_BRIGHT) {
+                        if (mDisplayPowerRequest.policy == DisplayPowerRequest.POLICY_BRIGHT
+                                || mDisplayPowerRequest.policy == DisplayPowerRequest.POLICY_VR) {
                             mUserActivitySummary = USER_ACTIVITY_SCREEN_BRIGHT;
                         } else if (mDisplayPowerRequest.policy == DisplayPowerRequest.POLICY_DIM) {
                             mUserActivitySummary = USER_ACTIVITY_SCREEN_DIM;
-- 
2.17.1

